### ISSUE TITLE
after update restoring to system timeout value doesn't work 

### ISSUE THREAD

**Issue by Simplenuity** [elastic-rock/KeepScreenOn#14]
system:
* Android 10
* Keep Screen On: at least versions 1.12.0 and 1.11.0

before update:
* in Keep Screen On both Options to restore timeout (low battery, screen off) activated
* icon shows up on status bar if Keep Screen On gets activated

after update:
* restoring to system timeout value doesn't work anymore
* icon doesn't show up on status bar

work around:
toggle both Options and app behaves as expected again

NB:
* a reboot is not solving the issue
* have not systematically tested whether status of Keep Screen On, i.e. being active or not, at time of update is making a difference

**Comment by elastic-rock** [comment]
Does reinstalling the app fix the issue?

**Comment by elastic-rock** [comment]
I had also encountered the issue with the tile being stuck after I updated the app on my own device. I did not however pay any special attention to it and proceeded with reinstalling the app with the assumption that it is just a problem on my end. It might be somehow related to the build workflow as there was [an issue with reproducible builds on f-droid](https://gitlab.com/fdroid/fdroiddata/-/commit/e15e1309fdf77121abb8e7d3918674713595551f#note_1608615147) but I have no idea what could have caused it.

Concerning the issue with the tiles being greyed out, I have not encountered it on my side.
